tools/v8_gypfiles/v8.gyp

```diff
@@ -1073,8 +1073,10 @@
             '<(V8_ROOT)/src/base/platform/platform-win32.cc',
             '<(V8_ROOT)/src/base/win32-headers.h',
           ],
-
-          'defines': ['_CRT_RAND_S'],  # for rand_s()
+          'defines': [
+            '_CRT_RAND_S',  # for rand_s()
+            '_WIN32_WINNT=0x0602' # For GetCurrentThreadStackLimits on Windows on Arm
```

Comment:
Given the discussions about allowing Node.js to run on unsupported Windows 7 (https://github.com/nodejs/node/issues/33034, https://github.com/nodejs/node/pull/33176) it seems prudent to point out that `_WIN32_WINNT=0x0602` is setting the "minimum operating system version your code supports" to Windows 8: https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019

cc FYI @nodejs/tsc @nodejs/platform-windows

Reply:
Since the use of these APIs is already protected by an Arm-specific macro, it's unlikely that this new define would break Windows 7 for now, but I don't have a Windows 7 device to validate my hunch. A possibility is to gate the definition further to only affect Arm processors (which will only run Windows 10). I'll try to push a new rev in the next day or two.

Reply:
Will address in the next push.